modules/images/src/index.js

```diff
@@ -2,7 +2,7 @@ export {default as ImageLoader} from './image-loader';
 export {default as ImageWriter} from './image-writer';
 
 // Parsed Image API
-export {isImageTypeSupported, getDefaultImageType} from './lib/parsed-image-api/image-type';
+export {isImageTypeSupported, getSupportedImageType} from './lib/parsed-image-api/image-type';
```

Comment:
This looks like it is reverting a change that I made for 2.1? Did it break something? I did keep the old name as a deprecated export?